src/api/app/views/daily_email_mailer/notifications.html.erb

```diff
@@ -0,0 +1,11 @@
+<% notifications.each do |notification| %>
```

Comment:
I agree to @ChrisBr since we're moving views to `haml` anyways, it makes sense to not write new `erb` templates. This also applys for the other views.